### DISCUSSION TITLE
[A-L] (17 aprile 2024) Esercizio 6

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/progettazione-di-algoritmi#51] [Esercitazioni A-L]
![image](https://github.com/sapienzastudentsnetwork/progettazione-di-algoritmi/assets/40034429/ddb70773-7696-48c8-8bdd-585dfea19e27)

**Comment by daromacarlo** [comment]
![immagine](https://github.com/user-attachments/assets/ee5a9670-154e-4be3-a58e-738e14ef2826)
